src/remappings.rs

```diff
@@ -12,6 +12,18 @@ const DAPPTOOLS_LIB_DIR: &str = "lib";
 const JS_CONTRACTS_DIR: &str = "contracts";
 const JS_LIB_DIR: &str = "node_modules";
 
+/// Convert all `\\` to `/` in a string path on Windows.
+#[cfg(target_os = "windows")]
+fn to_slash_path_preserve_trailing_slash(path: &str) -> String {
+    use path_slash::PathExt;
+    let has_trailing_slash = path.ends_with('/') || path.ends_with('\\');
+    let mut p = Path::new(path).to_slash_lossy().into_string();
```

Comment:
reverting the change fixes for windows

```suggestion
    let mut p = Path::new(path).to_slash_lossy().to_string();
```

Reply:
Oh my bad, I typod that. It's into_owned